File: class-继承/class.py
class Animal():
    def __init__(self, sex):
        self.sex = sex;
    def eat(self):
        print("eat");
    def getSex(self):
        print("sex", self.sex)

class Human(Animal):
    def __init__(self, sex):
        super().__init__(sex)
    def walk(self):
        print("walk")

human = Human(1);
human.getSex();
human.eat();
human.walk();

File: class-继承/class.js

const print = console.log

class Animal {

    constructor(sex) {
        this.sex = sex;
    }

    getSex() {
        print("sex", this.sex);
    }

    eat() {
        print("eat")
    }

}


class Human extends Animal {

    constructor(sex) {
        super(sex)
    }

    walk() {
        print("walk");
    }

}

var human = new Human(1, 0);
human.getSex();
human.eat();
human.walk();

